### PR TITLE
feat: add documentation to all public action on the beagle android

### LINKED ISSUE #975: Add documentation to all public action on the beagle android
## Use case
Public actions are not documented beagle android

## Proposal
Add javadoc documentation to all public Action at the beagle android.

### PR DESCRIPTION
**Pull request by luisoliveirazup**
### Related Issues
Closes #975 
 
### Description and Example
Public actions are not documented beagle android.

Expected: Add javadoc documentation to all public Action at the beagle android.

### Checklist

Please, check if these important points are met using `[x]`:

- [x] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [x] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [x] I am willing to follow-up on review comments in a timely manner.

[PR Guide]: https://github.com/ZupIT/beagle/blob/master/doc/contributing/pull_requests.md

### CODE REVIEW

**Review by diogooliveira-zup — DISMISSED** [review]

File: android/beagle/src/main/java/br/com/zup/beagle/android/action/Alert.kt [hunk 1]
```diff
@@ -27,6 +27,15 @@ import br.com.zup.beagle.android.utils.handleEvent
 import br.com.zup.beagle.android.view.ViewFactory
 import br.com.zup.beagle.android.widget.RootView
 
+/**
+ * will show dialogues natively, such as an error alert indicating alternative flows, business system errors and others.
```
Comment (matheusribeirozup) [inline android/beagle/src/main/java/br/com/zup/beagle/android/action/Alert.kt:31]:
```suggestion
 * This action will show dialogues natively, such as an error alert indicating alternative flows, business system errors and others.
```

File: android/beagle/src/main/java/br/com/zup/beagle/android/action/Condition.kt [hunk 2]
```diff
@@ -24,6 +24,14 @@ import br.com.zup.beagle.android.utils.evaluateExpression
 import br.com.zup.beagle.android.utils.handleEvent
 import br.com.zup.beagle.android.widget.RootView
 
+/**
+ * action to resolve condition and call onTrue if return true and onFalse if return is false.
```
Comment (matheusribeirozup) [inline android/beagle/src/main/java/br/com/zup/beagle/android/action/Condition.kt:28]:
```suggestion
 * Action to resolve condition and call onTrue if return true and onFalse if return is false.
```

File: android/beagle/src/main/java/br/com/zup/beagle/android/action/FormValidation.kt [hunk 3]
```diff
@@ -42,6 +50,13 @@ internal class FormValidation(
     }
 }
 
+/**
+ * class to define error.
```
Comment (matheusribeirozup) [inline android/beagle/src/main/java/br/com/zup/beagle/android/action/FormValidation.kt:54]:
```suggestion
 * Class to define error.
```

File: android/beagle/src/main/java/br/com/zup/beagle/android/action/Alert.kt (hunk 1 above)
Comment (luisoliveirazup) [reply android/beagle/src/main/java/br/com/zup/beagle/android/action/Alert.kt]: change done!

File: android/beagle/src/main/java/br/com/zup/beagle/android/action/Condition.kt (hunk 2 above)
Comment (luisoliveirazup) [reply android/beagle/src/main/java/br/com/zup/beagle/android/action/Condition.kt]: change done!

File: android/beagle/src/main/java/br/com/zup/beagle/android/action/Confirm.kt [hunk 4]
```diff
@@ -27,6 +27,17 @@ import br.com.zup.beagle.android.utils.handleEvent
 import br.com.zup.beagle.android.view.ViewFactory
 import br.com.zup.beagle.android.widget.RootView
 
+/**
+ * will show dialogues natively, such as an error alert indicating alternative flows, business system errors and others.
```
[the remark this answers is not in the document]
Comment (luisoliveirazup) [reply android/beagle/src/main/java/br/com/zup/beagle/android/action/Confirm.kt]: change done!

File: android/beagle/src/main/java/br/com/zup/beagle/android/action/FormValidation.kt (hunk 3 above)
Comment (luisoliveirazup) [reply android/beagle/src/main/java/br/com/zup/beagle/android/action/FormValidation.kt]: change done!

**Review by uziasferreirazup — APPROVED** [review]

**Review by matheusribeirozup — APPROVED** [review]

**Pull request merged**